r-package/grf/tests/testthat/data/test_causal_survival_forest_1.csv

```diff
@@ -0,0 +1,500 @@
+0.444564019146147
```

Comment:
It would be good to make a note somewhere about how this data was generated. And as a small comment, the file names could be more descriptive file names instead of using 1, 12, ...